FiveCalls/NotificationsService/NotificationService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  NotificationService.swift
+//  NotificationsService
+//
+//  Created by Nick O'Neill on 10/16/17.
+//  Copyright © 2017 5calls. All rights reserved.
+//
+
+import UserNotifications
+
+class NotificationService: UNNotificationServiceExtension {
+
+    var contentHandler: ((UNNotificationContent) -> Void)?
+    var bestAttemptContent: UNMutableNotificationContent?
+
+    override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
+        self.contentHandler = contentHandler
+        bestAttemptContent = (request.content.mutableCopy() as? UNMutableNotificationContent)
+        
+        if let bestAttemptContent = bestAttemptContent {
+            // Modify the notification content here...
+            bestAttemptContent.title = "\(bestAttemptContent.title) [modified]"
```

Comment:
This is just example code? Probably should remove.

Reply:
Oh yeah, for sure. I was thinking this was all the boilerplate needed without even looking at it.